packages/dx-react-scheduler/docs/reference/drag-drop-provider.md

```diff
@@ -72,6 +84,7 @@ Name | Properties | Description
 -----|------------|------------
 DragDropProvider.DraftAppointment | [DragDropProvider.DraftAppointmentProps](#dragdropproviderdraftappointmentprops) | A component that renders the appointment being dragged.
 DragDropProvider.SourceAppointment | [DragDropProvider.SourceAppointmentProps](#dragdropprovidersourceappointmentprops) | A component that renders a copy of the appointment being dragged in its previous location.
+DragDropProvider.Resize | [DragDropProvider.ResizeProps](#dragdropproviderresizeprops) | A component that renders a handler of resize appointment.
```

Comment:
DragDropProvider.ResizeComponent ?

Reply:
What do you mean?

Reply:
Oh, it's the 'Plugin Components' section. Everything is OK in this case.

Reply:
handler -> handle